src/components/IDO/IdoLabel1/IdoLabel1.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { ReactElement } from 'react'
```

Comment:
why did you divided right panel into 5 components, one for each item? one should be enough

Reply:
Thank you very much for your highly valuable comments, I almost completed fixing these mistakes for the next commit. Next afternoon you will probably have a new pull request.
I have a question, I will not push my next commit before your answer:
1) Would you like me to pass props directly to the Panel component? This will allow you to pass just five props in an object and all done. It can be ideal for presenting the data from the API.  {endSalePeriod : "", endGracePeriod:"", solAmountContibuted :"", estimatedTokenPrice:"", invariantForSale:"" }
2) Would you like me to pass props individually to the five separately organized label components? This will be also fine if we want to reach the parts of our panel or we want to change the design and present a new type of data.

I can do most things I saw in a figma file, but I also need to know what fits your purpose better. If you want something other than these two, please let me know.
best.

Reply:
first way should be good enough